Rocket/Rocket/Networking/RocketsManager.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  RocketsManager.swift
+//  Rocket
+//
+//  Created by Adela Mišicáková on 11.07.2022.
+//
+
+import Foundation
+import Combine
+
+class RocketsManager: ObservableObject {
+    
+    let apiService = ApiService()
```

Comment:
Ještě by dobrý použít Depedency Injection:

https://www.raywenderlich.com/14223279-dependency-injection-tutorial-for-ios-getting-started